website/src/components/external-link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import * as colors from '../colors';
+
+const linkStyles = css`
+  color: ${colors.light.primary};
+  text-decoration: none;
+
+  :hover {
+    text-decoration: underline;
+  }
+  :hover,
+  :active {
+    outline-width: 0;
+  }
+  :visited {
+    color: ${colors.light.primarySoft};
+  }
+`;
+
+const ExternalLink = props => (
```

Comment:
UIKit also has a link now which  has an `isExternal` prop. Maybe already enough?

Reply:
Yeah, I haven't test it yet because appkit still uses an "old" uikit version. I wanted to update it in a separate PR. Thanks for the reminder though!